R/read_txt_Renishaw.R

```diff
@@ -233,12 +233,11 @@ read_zip_Renishaw <- function(file = stop("filename is required"),
 
```

Comment:
@sangttruong, please, apply/use template #55 for unit testing `read_zip_Renishaw()`